datafusion/expr/src/udwf.rs

```diff
@@ -324,14 +317,8 @@ pub trait WindowUDFImpl: Debug + Send + Sync {
         hasher.finish()
     }
 
-    /// Allows customizing nullable of column for this window UDF.
-    ///
-    /// By default, the final result of evaluating the window UDF is
-    /// allowed to have null values. But if that is not the case then
-    /// it can be customized in the window UDF implementation.
-    fn nullable(&self) -> bool {
-        true
-    }
+    /// The [`Field`] of the final result of evaluating this window function.
```

Comment:
Could be useful to document here how the "name" for the returned field is supposed to be set :)

Reply:
Agreed. It's a great suggestion. I'll implement in a follow-up PR.

Thanks @Blizzara